content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/aging.rst

```diff
@@ -1,98 +1,227 @@
+
+.. image:: aging/after-val-method-change.png
+   :alt: Stock valuation table after switching from standard price to FIFO accounting.
+
+.. example::
+   The :guilabel:`Remaining Value` and :guilabel:`Remaining Quantity` fields are derived from what
+   occurs at the :abbr:`SVL (stock move layer)` level in Odoo and, as such, are better understood
+   with an example.
+
+   Frankie's Consignment Shop buys sweaters at the cost, or :guilabel:`Unit Value`, of `5.00`
+   dollars. For the first time, Frankie's purchases and receives a :guilabel:`Quantity` of `100.00`
+   sweaters in one stock move, then re-sells and delivers `-10.00` sweaters in a second stock move.
+
+   In the first stock move line item, the :guilabel:`Remaining Quantity` will change from `100.00`
+   to `90.00`, once the second stock move is recorded. This change reflects that, although 100
+   sweaters were originally purchased, only 90 of those sweaters remain in stock and should be
+   counted in the valuation. Similarly, the :guilabel:`Remaining Value` will drop from `$500.00` to
+   `$450.00`. The :guilabel:`Total Value` will remain at `$500.00`, regardless of subsequent
+   transactions.
+
+   On the other hand, the :guilabel:`Remaining Quantity` of the second stock move line item will be
+   recorded and remain at `0.00` because the quantity of `-10.00` was sold. In the system, because
+   the :abbr:`SVL (stock move layer)` was a sale, there is no stock left that needs to be valued
+   from that transaction.
+
+   .. image:: aging/remaining-val-quant.png
+      :alt: Remaining value and quantity are calculated based on :abbr:`SVLs (stock move layers)`.
+
+Change the valuation date
+-------------------------
+
+To see the valuation of stock moves at a specific date and time, click the :guilabel:`Valuation At
+Date` button, located in the top-left corner of the :guilabel:`Stock Valuation` page. The report
+will show the :guilabel:`Quantity` and :guilabel:`Total Value` of each stock move.
 
 .. note::
-   Records in the :guilabel:`Inventory Aging` report are *stock valuation layers* (SVLs),
-   representing product moves that impact stock valuation.
+   The :guilabel:`Remaining Quantity` and :guilabel:`Remaining Value` of the stock moves will *not*
+   be point-in-time for any dates chosen in the past. The stock moves shown when selecting a past
+   date will still display the *current on-hand quantity and value* of the products.
 
-   Inventory adjustments do **not** create |SVLs|; only items purchased from vendors do.
+.. example::
+   A business has 100 sofas in stock on January 1st and sells 20 of those sofas on February 1st. The
+   :guilabel:`Remaining Quantity` of the :abbr:`SVL (stock move layer)` will drop from `100.00` to
+   `70.00` on February 1st. If no other stock moves take place, and on February 1st, the
+   :guilabel:`Valuation at Date` is selected as January 1st, the :guilabel:`Remaining Quantity`
+   will still show as `70.00`.
```

Comment:
100 - 20 = 70 ? :thinking: 

https://x.com/ggellatly/status/1915171240010211416

Reply:
Oops! Addressing here https://github.com/odoo/documentation/pull/13060
Thank you